Specs/Scene/Cesium3DTileSpec.js

```diff
@@ -0,0 +1,149 @@
+/*global defineSuite*/
+defineSuite([
+        'Scene/Cesium3DTile',
+        'Core/Cartesian3',
+        'Core/defined',
+        'Core/OrientedBoundingBox',
+        'Core/Rectangle',
+        'Core/SphereOutlineGeometry',
+        'Specs/createContext',
+        'Specs/createFrameState'
+    ], function(
+        Cesium3DTile,
+        Cartesian3,
+        defined,
+        OrientedBoundingBox,
+        Rectangle,
+        SphereOutlineGeometry,
+        createContext,
+        createFrameState) {
+    "use strict";
+
+    describe('bounding volumes', function() {
+        var tileWithBoundingSphere = {
+            "geometricError" : 1,
+            "refine" : "replace",
+             "children" : [],
+             boundingVolume : {
+                 "sphere": [0.0, 0.0, 0.0, 5.0]
+             }
+        };
+
+        var tileWithContentsBoundingSphere = {
+            "geometricError" : 1,
+            "refine" : "replace",
+            "content" : {
+                "batchSize": 1,
+                "url" : "0/0.b3dm",
+                 boundingVolume : {
+                     "sphere": [0.0, 0.0, 0.0, 5.0]
+                 }
+             },
+             "children" : [],
+             boundingVolume : {
+                 "sphere": [0.0, 0.0, 0.0, 5.0]
+             }
+        };
+
+        it('can have a bounding sphere', function() {
+            var tile = new Cesium3DTile(undefined, '/some_url', tileWithBoundingSphere, undefined);
+            expect(defined(tile._tileBoundingSphere)).toEqual(true);
+            expect(tile._tileBoundingSphere.radius).toEqual(5.0);
+            expect(tile._tileBoundingSphere.center.equals(Cartesian3.ZERO)).toEqual(true);
+        });
+
+        it('can have a contents bounding sphere', function() {
+            var tile = new Cesium3DTile(undefined, '/some_url', tileWithContentsBoundingSphere, undefined);
+            expect(defined(tile._contentsBoundingSphere)).toEqual(true);
+            expect(tile._contentsBoundingSphere.radius).toEqual(5.0);
+            expect(tile._contentsBoundingSphere.center.equals(Cartesian3.ZERO)).toEqual(true);
```

Comment:
Change to `expect(tile._contentsBoundingSphere.center).toEqual(Cartesian3.ZERO);`